samples/SqlServerCacheSample/Program.cs

```diff
@@ -14,12 +14,10 @@ namespace SqlServerCacheSample
 {
     /// <summary>
     /// This sample requires setting up a Microsoft SQL Server based cache database.
-    /// 1. Install the command globally by doing "dnu commands install Microsoft.Extensions.Caching.SqlConfig". This
-    ///    installs a command called "sqlservercache".
+    /// 1. Install the command globally by installing the dotnet-sql-cache package.
```

Comment:
Ideally I would change the samples to have a `"tools"` section illustrating the tool's usage with dotnet, but that won't work now since tool references must be package references i.e. not project references. Once the first package is publish, we could change the samples to use `dotnet-sql-cache` as a tool.